partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnBetween.kt

```diff
@@ -1,230 +1,152 @@
-// ktlint-disable filename
-@file:Suppress("ClassName")
-
 package org.partiql.spi.function.builtins
 
+import org.partiql.spi.function.Fn
+import org.partiql.spi.function.FnOverload
+import org.partiql.spi.function.Function.instance
 import org.partiql.spi.function.Parameter
+import org.partiql.spi.function.RoutineOverloadSignature
+import org.partiql.spi.function.builtins.FnGt.getNumber
```

Comment:
I think this is an incorrect import?

Reply:
Yea, not too sure what happened with this import. Seems like it was importing the `getNumber` through the `FnGt` object which extends `DiadicComparisonOperator`. Fixed in [`3cfeb1d` (#1755)](https://github.com/partiql/partiql-lang-kotlin/pull/1755/commits/3cfeb1df6b0ec2dde306c075bcf9f12798e5eb0f).